README.md

```diff
@@ -63,6 +63,7 @@
 ### Surfshark
 
 - Pick the [region](https://github.com/qdm12/private-internet-access-docker/wiki/Surfshark) or a multi hop region name
+- Fetch the `username` and `password` from the bottom of the [manual setup](https://account.surfshark.com/setup/manual) page.

```

Comment:
```suggestion
```

I would keep that information in the environment variables table where it's useful (as you did as well 👍 )

Reply:
As you wish :) My thought was that having it in multiple places increases the odds of people finding it (and decreases the amount of created issues)

Reply:
Agreed, although that's a list of features (if I recall correctly) and then I'd have to add it for other providers.

Plus, fun fact, the Docker Hub API limits readmes to 25KB which I've reached with this repository so it would no longer sync the readme :laughing:  So I'm saving bytes now haha